journal-of-water-sanitation-and-hygiene-for-development.csl

```diff
@@ -0,0 +1,604 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB" demote-non-dropping-particle="never">
+  <info>
+    <title>Journal of Water Sanitation and Hygiene for Development</title>
+    <title-short>JWaSHDev</title-short>
+    <id>http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development</id>
+    <link href="http://www.zotero.org/styles/journal-of-water-sanitation-and-hygiene-for-development" rel="self" />
+    <link href="http://www.zotero.org/styles/apa-old-doi-prefix" rel="template" />
```

Comment:
@damnation333, I don't see any indication in http://washdev.iwaponline.com/content/instructions-authors that this journal requires "doi:" instead of "https://doi.org/". Since the latter is the preferred form nowadays (because it creates an URL that can be directly accessed), I wouldn't hesitate to push it unless the style guide explicitly wants something else, even if published articles use an outdated format like "doi:".